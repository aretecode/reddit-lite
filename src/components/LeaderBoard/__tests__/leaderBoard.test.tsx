import * as React from 'react'
import { render } from 'react-testing-library'
import { LeaderBoard } from '../'
import * as exported from '../'

describe('LeaderBoard', () => {
  it('should match export snapshot', () => {
    expect(exported).toMatchSnapshot()
  })
  it('should render without errors', () => {
    const { container } = render(<LeaderBoard />)
    expect(container).toMatchSnapshot()
  })
})
