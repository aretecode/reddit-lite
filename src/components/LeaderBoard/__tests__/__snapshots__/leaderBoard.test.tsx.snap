// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeaderBoard should match export snapshot 1`] = `
Object {
  "LeaderBoard": [Function],
  "StyledLeaderBoard": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "componentStyle": ComponentStyle {
      "componentId": "sc-bdVaJa",
      "isStatic": true,
      "rules": Array [
        "
  display: flex;
  justify-content: center;
  align-items: center;
  position: sticky;
  background-color: rgb(66, 173, 240);
  border-bottom: 1px solid #e9f5fd;
  height: 50px;
",
      ],
    },
    "displayName": "styled.header",
    "foldedComponentIds": Array [],
    "render": [Function],
    "styledComponentId": "sc-bdVaJa",
    "target": "header",
    "toString": [Function],
    "warnTooManyClasses": [Function],
    "withComponent": [Function],
  },
  "default": [Function],
}
`;

exports[`LeaderBoard should render without errors 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: -webkit-sticky;
  position: sticky;
  background-color: rgb(66,173,240);
  border-bottom: 1px solid #e9f5fd;
  height: 50px;
}

<div>
  <header
    class="c0"
  />
</div>
`;
